script.js

```diff
@@ -0,0 +1,94 @@
+const myBooks = [];
+const addBtn = document.getElementById('add-btn');
+const listBooks = document.getElementById('list-books');
+
+// load books from localStorage
+const storedBooks = localStorage.getItem('myBooks');
+if (storedBooks) {
+  myBooks.push(...JSON.parse(storedBooks)); // Parse the storedBooks JSON string
+}
+
+function showBooks() {
+  listBooks.innerHTML = '';
+
+  // Show all books in the myBooks array using forEach() loop
+  myBooks.forEach((myBook) => {
+    const myBookDiv = document.createElement('div'); // Create a new div for each book
+
+    // Create h4 element for title and append textContent
+    const bookTitle = document.createElement('h4');
+    bookTitle.textContent = myBook.bookTitle;
+
+    // Create h4 element for author and append textContent
+    const bookAuthor = document.createElement('h4');
+    bookAuthor.textContent = myBook.bookAuthor;
+
+    // Create remove button
+    const removeBtn = document.createElement('button');
+    removeBtn.textContent = 'Remove';
+
+    // Add event listener to remove button
+    removeBtn.addEventListener('click', () => {
+      // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] `required`
- [ ] Please enable the disabled `eslint` because it defeats the purpose of adding linters. 🥂

Reply:
Hello @mwanawabangona , I need your insight here, if I enable the requested change, I will get a linter error, 
`'removeBook' was used before it was defined.eslint`. This does not affect the program execution at all. How should I go about this? Kindly advise. I have tried refactoring my code by re-organising the functions but still run into the same problem.